File: 0225.implement_stack_using_queues/solution.java
import java.util.ArrayList;

class MyStack{
    private ArrayList<Integer> stack;

    public MyStack(){
        stack = new ArrayList<>();
    }

    public void push(int x){
        stack.add(x);
    }

    public int pop(){
        if (!this.empty()){
            return stack.remove(stack.size() - 1);
        }
        return -1;
    }

    public int top(){
        if (!this.empty()){
            return stack.get(stack.size() - 1);
        }
        return -1;
    }

    public boolean empty(){
        return stack.isEmpty();
    }
}

File: 0225.implement_stack_using_queues/solution.py
class MyStack(object):
    def __init__(self):
        self.ans = []

    def push(self, x):
        self.ans.append(x)

    def pop(self):
        return self.ans.pop()

    def top(self):
        return self.ans[-1]

    def empty(self):
        return self.ans == []
